packages/workbox-core/src/_private/getFriendlyURL.ts

```diff
@@ -11,7 +11,7 @@ import '../_version.js';
 const getFriendlyURL = (url: URL | string) => {
   const urlObj = new URL(String(url), location.href);
   if (urlObj.origin === location.origin) {
-    return urlObj.pathname;
+    return urlObj.pathname + urlObj.hash;
```

Comment:
I wonder if it makes sense to keep the `search` portion as well? Based on that issue I think someone could run into a similar problem with a stray query param.

Reply:
Yeah, good point.

Reply:
I'm just stripping the leading origin now, if it's a same-origin URL.